pages/docs/tutorials/kafka/bindings-with-kafka.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Kafka bindings
+description: Learn how to configure Kafka bindings in your AsyncAPI document.
+weight: 300
+---
+
+## Introduction
+
+In a previous tutorial, you learned how to manage schemas with a schema registry. This tutorial teaches you how Kafka bindings function by defining Kafka messages and expanding your AsyncAPI document with protocol-specific details.
+
+## Background context
+
+Bindings are essential for event-driven applications because they provide protocol-specific details, abstracting the complexities of message handling from your application's core logic. They enhance the API's clarity and usability by offering setup options and context for different protocols. Bindings include details like the topics your application reads from or writes to, message formatting, and rules for interacting with multiple data or messages.
+
+In an AsyncAPI document, bindings can be added to various sections like servers, channels, or messages. They contain protocol-specific details unique to each protocol. Binding definitions let you specify functionalities specific to the protocol, which are not covered by AsyncAPI's core features.
+
+You can configure several objects using Kafka bindings. However, for the scope of this tutorial, you will focus on four levels of bindings: [server bindings](https://github.com/asyncapi/bindings/tree/master/kafka#server-binding-object), [operations binding](https://github.com/asyncapi/bindings/tree/master/kafka#operation-binding-object), [channel bindings](https://github.com/asyncapi/bindings/tree/master/kafka#channel-binding-object) and [message bindings](https://github.com/asyncapi/bindings/tree/master/kafka#message-binding-object).
+
+Using the code snippets from the previous tutorial, where you learned [how to manage Avro schemas using a centralized schema registry that enables you to share schemas across multiple applications](pages/docs/tutorials/kafka/managing-schemas-using-schema-registry), you will add configurations for server, operations, channel, and message bindings.
+
+## Add server bindings
+
+Server bindings provide protocol-specific configuration details for connecting and interacting with a server. For server bindings, we will add three fields: 
```

Comment:
I think I left that comment some time ago already

we need intros to these snippets

in this one for example you add schema registry details, why?

for example here you explain: server binding for example allows you to specify for a given server, where is related schema registry located. Schema registry is a repository for managing and validating messages' schemas. To learn more about schema registry read https://www.asyncapi.com/docs/guides/message-validation#schema-registry-validation

you know what I mean.

you do not have to go through each property, they are well explained in bindings https://github.com/asyncapi/bindings/tree/master/kafka

but an intro makes sense - so with example they understand the concept behind it